tests/libcxx/expected_results.txt

```diff
@@ -817,16 +982,16 @@ std/re/re.const/re.matchflag/match_prev_avail.pass.cpp FAIL
 std/input.output/filesystems/class.path/path.member/path.charconv.pass.cpp FAIL
 
 # Not yet analyzed. Probably ADL shenanigans.
+std/algorithms/robust_against_adl.compile.pass.cpp FAIL
+std/strings/basic.string/string.modifiers/robust_against_adl.pass.cpp FAIL
 std/thread/thread.threads/thread.thread.class/thread.thread.constr/robust_against_adl.pass.cpp FAIL
 std/utilities/function.objects/func.wrap/func.wrap.func/robust_against_adl.pass.cpp FAIL
 std/utilities/function.objects/refwrap/refwrap.invoke/robust_against_adl.pass.cpp FAIL
 std/utilities/variant/variant.visit/robust_against_adl.pass.cpp FAIL
 
-# Not yet analyzed. Probably MSVC bug.
+# Not yet analyzed. Possibly C1XX constexpr bug.
 std/utilities/function.objects/func.invoke/invoke_constexpr.pass.cpp:0 FAIL
-
-# Not yet analyzed. Probably name mangling bug.
-std/utilities/function.objects/func.wrap/func.wrap.func/noncopyable_return_type.pass.cpp FAIL
+std/ranges/range.adaptors/range.join.view/iterator/increment.pass.cpp:0 FAIL

```

Comment:
Unnecessary divergence - `skipped_tests.txt` correctly lacks this line, as it appears above in the section

```
# Likely STL bug in `join_view::_Iterator`: constexpr weirdness
```